cmd/experimental/kueue-viz/frontend/start.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Copyright The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generate env.js with runtime environment variables
+cat << EOF > ./build/env.js
+/*
```

Comment:
Why not load license from https://github.com/kubernetes-sigs/kueue/blob/main/hack/boilerplate.go.txt?

Reply:
That makes sense. Thank you